Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook

```diff
@@ -293,7 +334,96 @@
           }
         ]
       },
+      "conditionalVisibility": {
+        "parameterName": "tabName",
+        "comparison": "isEqualTo",
+        "value": "limitsAndUsage"
+      },
       "name": "groupLimitsAndUsage"
+    },
+	{
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": ""
+            },
+            "name": "infoUnusedMetrics"
+          },
+		  {
+			"type": 9,
+			"content": {
+			  "version": "KqlParameterItem/1.0",
+			  "parameters": [
+				{
+				  "id": "05607eff-fcbc-40b8-8eda-e36c4b90a6f7",
+				  "version": "KqlParameterItem/1.0",
+				  "name": "notUsedInDays",
+				  "label": "Not Used In Days",
+				  "type": 2,
+				  "isRequired": true,
+				  "query": "{\"version\":\"1.0.0\",\"content\":\"[\\\"0d\\\", \\\"30d\\\", \\\"60d\\\", \\\"90d\\\"]\",\"transformers\":null}",

```

Comment:
Sorry the `0d` option was just for testing - given that Praval's AMW looks to have reasonable data I think we can remove this even at this point so we don't forget about this later (or if needed for testing, if you could create a tracking task)